runtime/kubeadm.go

```diff
@@ -45,7 +45,13 @@ func (d *Default) templateFromContent(templateContent string) ([]byte, error) {
 	}
 
 	var envMap = make(map[string]interface{})
-	envMap[CertSANS] = d.APIServerCertSANs
+	sans := []string{"127.0.0.1"}
+	sans = utils.AppendIPList(sans, []string{d.APIServer})
+	sans = utils.AppendIPList(sans, d.Masters)
```

Comment:
d.Masters maybe include port link this: 192.168.0.2:2202 need use utils.GetHostIPSlice(d.Masters)